docs/installation-docker.md

```diff
@@ -0,0 +1,31 @@
+# Installation
+
+## Requirements
+
+docker + docker-compose
+
+## Let's begin
+
+Before we can run application with docker we need to install composer dependencies.
+
+```bash
+docker run -v `pwd`:/var/www -w /var/www prooph/composer:7.1 install
+```
```

Comment:
Why not mounting to `/app`, no workdir definition needed?

Reply:
We can use `/app` without working directory definition. Used docker image by default set it to `/app`. But if we want to keep same as here https://github.com/codeliner/php-ddd-cargo-sample/blob/master/docker-compose.yml#L18 we should `/var/www` that also need `-w` option because composer will trigger install in `/app`.

Reply:
In both cases (/app and /var/www) the host working directory is mounted so it makes no real difference if /app is used in one container and /var/www in the other, but I agree that it is easier to understand how things work if we always mount and set the same directory.

Reply:
@codeliner @sandrokeil instructions added https://github.com/codeliner/php-ddd-cargo-sample/pull/48/files#diff-8f913077a8342cf43a8dfa290e89df3bR8